servers/lib/server/index.coffee

```diff
@@ -120,12 +118,12 @@ app.get  '/-/api/ssotoken/login'                 , require './handlers/api/ssoto
 app.get  '/-/api/logs'                           , require './handlers/api/logs'
 app.post '/-/api/gitlab', bodyParser.json()      , require './handlers/api/gitlab'
 app.get  '/-/image/cache'                        , require './image_cache'
-app.get  '/-/oauth/github/callback'              , require './github_callback'
+# app.get  '/-/oauth/github/callback'              , require './github_callback'
```

Comment:
I'm planning to get them back pretty soon, so I kept all the code as is for now. I can do that in the future PR if I won't be able to make it happen as I planned, ok?